packages/core/src/mask/MaskSystem.js

```diff
@@ -166,79 +243,17 @@ export class MaskSystem extends System
     }
 
     /**
-     * Applies the Mask and adds it to the current filter stack.
-     *
-     * @param {PIXI.Sprite|PIXI.Graphics} maskData - The masking data.
-     */
-    pushStencilMask(maskData)
-    {
-        this.renderer.batch.flush();
-        this.renderer.stencil.pushStencil(maskData);
-    }
-
-    /**
-     * Removes the last filter from the filter stack and doesn't return it.
-     *
+     * Checks mask balance after frame rendering
      */
-    popStencilMask()
+    postrender()
     {
-        // this.renderer.currentRenderer.stop();
-        this.renderer.stencil.popStencil();
-    }
-
-    /**
-     *
-     * @param {PIXI.DisplayObject} target - Display Object to push the mask to
-     * @param {PIXI.Graphics} maskData - The masking data.
-     */
-    pushScissorMask(target, maskData)
-    {
-        maskData.renderable = true;
-
-        const bounds = maskData.getBounds(true);
-
-        maskData.renderable = false;
-
-        this.renderer.gl.enable(this.renderer.gl.SCISSOR_TEST);
-
-        const resolution = this.renderer.resolution;
-
-        let y;
-
-        if (this.renderer.renderingToScreen)
+        if (this.errorCounter > 0)
```

Comment:
Yo! Do we need the error count stuff? Could it be covered in a unit test instead?

Reply:
The thing is, I saw people are writing custom versions of container.renderAdvanced() method to add more masks. I want to cover all push-pops check later, "filter.pop()" too. I believe in that case small "if" in code can save hours of debugging.